DigitalLearningSolutions.Web/Views/SuperAdmin/Centres/_CentreCard.cshtml

```diff
@@ -0,0 +1,90 @@
+﻿@using DigitalLearningSolutions.Web.ViewModels.SuperAdmin.Centres
+@model CentreSummaryViewModel
+
+<div class="searchable-element nhsuk-expander-group" id="@Model.CentreId-card">
+  <details class="nhsuk-details nhsuk-expander nhsuk-u-margin-bottom-0">
+    <summary class="nhsuk-details__summary">
+      <span class="nhsuk-details__summary-text searchable-element-title" id="@Model.CentreId-name" name="centre-name">
+        @Model.CentreName (@Model.CentreId)
+      </span>
+    </summary>
+
+    <div class="nhsuk-details__text">
+      <dl class="nhsuk-summary-list">
+
+        <div class="nhsuk-summary-list__row">
+        <dt class="nhsuk-summary-list__key">
+          Centre id
+        </dt>
+        <dd class="nhsuk-summary-list__value">
+          @Model.CentreId
+        </dd>
+    </div>
+
+    <div class="nhsuk-summary-list__row">
+      <dt class="nhsuk-summary-list__key">
+        Centre type
+      </dt>
+      <dd class="nhsuk-summary-list__value">
+        @Model.CentreType
+      </dd>
+    </div>
+
+    <div class="nhsuk-summary-list__row">
+      <dt class="nhsuk-summary-list__key">
+        Region
+      </dt>
+      <dd class="nhsuk-summary-list__value">
+        @Model.RegionName
+      </dd>
+    </div>
+
+    <div class="nhsuk-summary-list__row">
+      <dt class="nhsuk-summary-list__key">
+        Contact
+      </dt>
+      <dd class="nhsuk-summary-list__value">
+        @Model.ContactForename @Model.ContactSurname
+      </dd>
+    </div>
+
+    <div class="nhsuk-summary-list__row">
+      <dt class="nhsuk-summary-list__key">
+        Contact email
+      </dt>
+      <dd class="nhsuk-summary-list__value">
+        @Model.ContactEmail

```

Comment:
We generally use dashes when these strings are null. I guess we should do the same here.

Reply:
Done.